sourcemash/api/categories.py

```diff
@@ -7,6 +7,12 @@
 from sqlalchemy import func
 from collections import Counter
 
+class CategoryAPI(Resource):
```

Comment:
Why did you make this API call / change from "model" to "title" as part of this PR? Just curious.